source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -64,12 +64,19 @@ export const getPullRequestsAssociatedWithBranch = cache.function(async (): Prom
 
 // TODO: Replace this with `State--${prInfo.state.toLowerCase()}` GHE #4202
```

Comment:
Just do the TODO were anyways going to do it in a month

Reply:
It doesn't look nice:

```diff
- className={`State ${stateClass[prInfo.state]} State--small ml-1`}
+ className={`State ${prInfo.state === 'MERGED' ? '' : `State--${prInfo.state.toLowerCase()}`} State--small ml-1`}
```

Reply:
> ```diff
> prInfo.state === 'MERGED' ?
> ```

Why do you need this?

Reply:
Did you mean to write `draft`?

If thats the case nothing will happen if you add a non existing class

Reply:
> Did you mean to write `draft`?

Yep

> If thats the case nothing will happen if you add a non existing class

I just noticed it does exist and behaves the same:

![image](https://user-images.githubusercontent.com/44045911/132724876-b860dcbe-6f32-48f2-acee-858fe5f8f942.png)

So let's do it